src/entity.js

```diff
@@ -178,7 +178,7 @@ function Key(options) {
   this.namespace = options.namespace;
 
```

Comment:
I think we could just add this line around here:

```js
options.path = [].slice.call(options.path);
```

Reply:
That is indeed a lot better; I'll make this change 👍